workflow/scripts/plot_score.R

```diff
@@ -0,0 +1,57 @@
+#This is a script for plotting a heatmap overview of aMeta authentication scores.
+#Run this script as:
+#Rscript plot_score.R in_dir out_dir
+
+#in_dir<-"aMeta/results/AUTHENTICATION"
+#out_dir<-"aMeta/results"
+
+args = commandArgs(trailingOnly=TRUE)
+in_dir<-as.character(args[1])
+out_dir<-as.character(args[2])
+
+samples<-list.files(in_dir)
+
+score_per_sample<-list()
+for(i in samples)
+{
+  taxa<-list.files(paste0(in_dir,"/",i))
```

Comment:
I would suggest using `file.path` instead of `paste0` for constructing file paths as this makes it system-agnostic: 

`file.path(in_dir, i)`

Reply:
file.path will automatically know which file separator to use